packages/sdk-client/src/api/api-client.ts

```diff
@@ -5,8 +5,13 @@ import FormData = require('form-data');
 
 export enum PaginationEnum {
```

Comment:
May be having such list + naming + comment + dedicated field names variation processing is highlighting a design to be improved: is it possible to have something like
- a common interface definition/abstraction (avoiding dedicated payload names)
- each domain is implementing it own real data to data abstraction service
 
And, because at domain level, it won't "hurt" to have domain logic moved  from `generic client` to `dedicated domain`